src/morello-install/README.md

```diff
@@ -670,28 +677,27 @@ Otherwise, press Enter to complete the installation:
 If desired, select Yes and press Enter to drop to a shell before reboot.
 Otherwise, select No and press Enter.
 ```
-                   ┌───────Manual Configuration──────────┐
-                   │ The installation is now finished.   │
-                   │ Before exiting the installer, would │
-                   │ you like to open a shell in the new │
-                   │ system to make any final manual     │
-                   │ modifications?                      │
-                   ├─────────────────────────────────────┤
-                   │         < Yes >     < No  >         │
-                   └─────────────────────────────────────┘
+                 ┌──────────┤Manual Configuration├──────────┐
+                 │ The installation is now finished. Before │
+                 │ exiting the installer, would you like to │
+                 │ open a shell in the new system to make   │
+                 │ any final manual modifications?          │
+                 ├──────────────────────────────────────────┤
+                 │           [ Yes  ]     [  No  ]          │
+                 └──────────────────────────────────────────┘
 ```
 
 ### Complete
 
 Select Reboot and press Enter:
 ```
-                     ┌────────────┤Complete├─────────────┐
-                     │ Installation of FreeBSD complete! │
-                     │ Would you like to reboot into the │
-                     │ installed system now?             │
-                     ├───────────────────────────────────┤
-                     │ [ Reboot ] [Shutdown] [Live CD ]  │
-                     └───────────────────────────────────┘
+                  ┌───────────────┤Complete├────────────────┐
+                  │ Installation of FreeBSD complete! Would │
+                  │ you like to reboot into the installed   │
+                  │ system now?                             │
+                  ├─────────────────────────────────────────┤
+                  │[  Reboot   ] [ Shutdown  ] [Live System]│
+                  └─────────────────────────────────────────┘
```

Comment:
The "Logging in" section below needs an updated, but that depends on the final release image.